ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_core.cc

```diff
@@ -93,52 +89,57 @@ TFCore::getModelPath ()
  * @note	the model will be loaded
  * @return 0 if OK. non-zero if error.
  *        -1 if the pb file is not loaded.
- *        -2 if the input properties is different with model.
+ *        -2 if the input properties are different with model.
  *        -3 if the Tensorflow session is not created.
  */
 int
 TFCore::loadModel ()
 {
+  g_message ("Start to load model [%s]", model_path);
```

Comment:
Please don't write messages unconditionally for "normal operations".

Reply:
logs about start/end to load model are removed